src/structures/interfaces/InteractionResponses.js

```diff
@@ -69,18 +69,19 @@ class InteractionResponses {
 
   /**
    * Creates a reply to this interaction.
+   * <info>By default `void`, provide the `fetchReply` option to fetch the bot's response.</info>
```

Comment:
```suggest
   * <info>By default, the return type is `void`. Provide the `fetchReply` option to fetch the bot's response with the return type being `Message`</info>
```

Reply:
^

Reply:
don't remove the period and we can talk about it

Reply:
On second thought, listing all the response types doesn't add any benefit to the info block. that's what the function's return type is meant to provide. Listing `void` and `Message` would be wrong, as it does return `Promise<*>` for either case. (+ the possibility to return an `APIMessage` instead. Covering all this takes up a lot of space and probably results in less people reading the info.